docs/guides/assembly.xml

```diff
@@ -13,7 +13,7 @@
             </includes>
         </fileSet>
         <fileSet>
-            <directory>${project.basedir}/src/main/server</directory>
+            <directory>${project.basedir}/server</directory>
```

Comment:
I sneaked in a little fix for keycloak-web. It turned out the pinned guides actually didn't work there.

CC: @ahus1

Reply:
Ooops! Thanks for fixing this. Do we plan another KC minor release, and should this be backported?

Reply:
I think nobody really noticed maybe. :) So perhaps no need for a backport?

CC @stianst